components/WhatDrivesUs/WhatDrivesUs.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { useTheme } from "@mui/system";
+import { Content } from "../../layout/Content";
+import { Grid } from "@mui/material";
+import { SectionTitle } from "../SectionTitle";
+import { LinearGradient } from "../LinearGradient";
+import { DrivingSection } from "../DrivingSection/DrivingSection";
+
+export const WhatDrivesUs = () => {
+  const theme = useTheme();
+
+  return (
+    <Content
+      pt={12}
+      pb={{ xs: 11, md: 16 }}
+      sx={{ backgroundColor: theme.palette.common.black }}
+    >
+      <Grid
+        item
+        container
+        direction="column"
+        alignItems={{ xs: "start", md: "center" }}
```

Comment:
personally I don't like that styling as propses, I think it creates a mess, but if you all want to do it that way so let's keep it